frontend/src/components/Overview.tsx

```diff
@@ -82,18 +82,18 @@ export const OverviewContent: FC<OverviewContentProps> = ({
 
 /**
  * Renders a <Overview /> component, which can be used in task, client etc. overviews.
- * @param {object} props Component props
- * @param {string} props.backHref Href for the back button's link
```

Comment:
Good change. At the time of writing this I was unfamiliar how JSDoc should be written for React & typescript, but thinking back the type annotations are probably unnecessary.